cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
Will this break if freertos or the arduino Scheduler library is used in combination with this library?